clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -652,28 +652,28 @@ and the instruction fetch ordinal.
 \code
 $ $ bin64/drrun -t drcachesim -simulator_type view -indir drmemtrace.*.dir -sim_refs 20
 Output format:
-<record#> <instr#>: T<tid> <record details>
+<--record#-> <--instr#->: T<tid> <record details>
```

Comment:
Hmm the thing that jumps out now, since there is clear effort to align the first two columns, is that `<tid>` doesn't line up.  Should we impose a consistent width and add the `-`'s here?

Reply:
I don't think a separate issue is needed: the same issue in the tracker can cover it.  It is quite related to this change in that the headers looked ok before but now they look inconsistent with the deliberate "---" showing the columns but not being there for the tid.